website/sidebars.json

```diff
@@ -18,7 +18,8 @@
       "native-modules-marshalling-data",
       "native-modules-async",
       "native-modules-jsvalue",
-      "native-modules-advanced"
+      "native-modules-advanced",
+      "native-modules-nuget"

```

Comment:
we'll want this to go into the Experimental tab (added in https://github.com/microsoft/react-native-windows-samples/pull/214)

Reply:
@kikisaints: Apologies for the merge conflict, your PR had experimental at the top, so I modelled it after that.